templates/web/base/waste/summary.html

```diff
@@ -17,7 +17,7 @@ <h1 class="govuk-heading-xl">[% title %]</h1>
     <dd class="govuk-summary-list__value">
     </dd>
     <dd class="govuk-summary-list__actions">
-    <form method="post">
+    <form method="post" id="change_answers">
```

Comment:
If you didn't want to add a special ID just for test and you know what `step1` is, I think submit_form_ok({ with_fields => { goto => [step1 value] } }) would also work.